apps/landing/components/Tutors/index.tsx

```diff
@@ -1,34 +1,33 @@
+"use client";
+
 import Content from "@/components/Tutors/Content";
-import { api } from "@/lib/api";
-import { formatMessage } from "@/lib/intl";
 import { Typography } from "@litespace/ui/Typography";
 import React from "react";
 import InViewTrack from "@/components/Common/InViewTrack";
-import { isEmpty } from "lodash";
-
-export const Tutors: React.FC = async () => {
-  const intl = await formatMessage();
-  const tutors = await api.user
-    .findOnboardedTutors({
-      page: 1,
-      size: 4,
-    })
-    .catch(() => {
-      return { list: [], total: 0 };
-    });
+import { useFormatMessage } from "@/hooks/intl";
+import { ITutor } from "@litespace/types";
 
-  if (isEmpty(tutors.list)) return null;
+export const Tutors: React.FC<{
+  tutors: ITutor.FindOnboardedTutorsApiResponse;
+}> = ({ tutors }) => {
+  const intl = useFormatMessage();
 
   return (
-    <div className="bg-secondary-50 max-w-screen">
+    <div className="bg-natural-0 max-w-screen">
```

Comment:
`bg-natural-0` is not a color. 

```suggestion
    <div className="max-w-screen">
```